client/contracts/truffle/contracts/utils/NameHash.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity 0.4.24;
+
+import "./Strings.sol";
+
+
+/**
+ * Namehash Solidity implementation.
+ */
+library NameHash
+ {
+    using strings for *;
+
+
+    /**
+     * Native namehash algorighm implementation in Solidity.
```

Comment:
algorithm*

Reply:
That should really be caught by the Go test.